community/urls.py

```diff
@@ -63,4 +64,6 @@ def get_index():
         distill_func=get_index,
         distill_file='log/index.html',
     ),
+    url(r'^data/', include('data.urls')),
```

Comment:
you need to use `distill_url`  ;-)

Reply:
damn! never used that, let me read their docs then :)